### ISSUE TITLE
Improvements on Transforms Documentation

### ISSUE THREAD

**Issue by MikaelahD13** [CDCgov/prime-reportstream#14820]
## Problem statement

Identify potential improvements to our documentation on creating transforms

## Acceptance criteria
- [x] Identify gaps and potential improvements to our documentation on creating transforms noted in this ticket
- [x] Reviewed and aligned with Onboarding Engineers

**Comment by chris-kuryak** [comment]
@victor-chaparro Can we add this as an agenda item to Engr sync this Friday?

**Comment by victor-chaparro** [comment]
- Generic Examples for filters  Covid Pipeline vs UP - FHIR path
-- Regex match, does not match, has valid data for
- Examples of how to implement Covid Pipeline settings as receiver transforms
- How to filter antigen vs antibody?
- Examples for how to use look up tables in transforms
- FHIR enrichment transforms vs Receiver transforms

**Comment by victor-chaparro** [comment]
@JessicaWNava go through questions and see what could be added as documentation

**Comment by chris-kuryak** [comment]
@victor-chaparro What is the status on this one?

**Comment by chris-kuryak** [comment]
Antigen / Antibody documentation still blocked by Platform needing to finish the feature. Otherwise, all other ideas can be considered for a future sprint.

These are nice to haves, not need to haves.
